src/aws/builder.rs

```diff
@@ -1526,7 +1526,8 @@ mod tests {
             .with_region("region")
             .with_bucket_name("bucket_name")
             .with_allow_http(true)
-            .with_proxy_url("asdf://example.com")
+            // use invalid url
```

Comment:
I suspect, but don't know, that this change is required by 
- https://github.com/seanmonstar/reqwest/pull/2681

the test expects the proxy config to fail, but after the reqwest upgrade it was passing